arkh/arkh.json

```diff
@@ -18,13 +18,24 @@
     "fees": {
         "fee_tokens": [
             {
-                "denom": "uarkh",
+                "denom": "arkh",
```

Comment:
this needs to be the base_denom, which I assume is `uarkh`.

Speaking of which, can you please add an assetlist as well?

See the Osmosis example:  https://github.com/cosmos/chain-registry/blob/master/osmosis/assetlist.json